dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/CouponTest.php

```diff
@@ -92,14 +92,16 @@ public function testApplyCouponTwice()
      */
     public function testApplyCouponToCartWithNoItems()
     {
-        $this->markTestIncomplete('https://github.com/magento/graphql-ce/issues/191');
         $couponCode = '2?ds5!2d';
 
         $this->quoteResource->load($this->quote, 'test_order_1', 'reserved_order_id');
-        $maskedQuoteId = $this->quoteIdToMaskedId->execute((int)$this->quote->getId());
+        $cartId = (int)$this->quote->getId();
+        $maskedQuoteId = $this->quoteIdToMaskedId->execute($cartId);
         $query = $this->prepareAddCouponRequestQuery($maskedQuoteId, $couponCode);
 
-        self::expectExceptionMessageRegExp('/Cart doesn\'t contain products/');
+        self::expectExceptionMessage(
+            str_replace('$cartId', $cartId, 'The "$cartId" Cart doesn\'t contain products.')
```

Comment:
Using of `str_replace` is overhead for this case, you can use just concatenation

Reply:
used concatenation instead str_replace